index.js

```diff
@@ -156,27 +177,37 @@ module.exports = (bookshelf, options) => {
      * @return {Number} The number of rows affected by the restoration
      */
     history (model, patch, operation, transacting) {
-      let fields = model.historyOptions.fields
-      let execute = transacting => {
-        let forge = {}
+      const fields = model.historyOptions.fields
+      const execute = transacting => {
+        const forge = {}
         forge[fields.resource_id] = model.id
         forge[fields.resource_type] = model.tableName
 
+        if (!transacting) {
```

Comment:
@bsiddiqui is this the right place to put the warning?

Reply:
yeah this works